tests/puppeteer/campaigns/functional/BO/catalog/stocks/02_updateQuantity.js

```diff
@@ -0,0 +1,97 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+const {Products} = require('@data/demo/products');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const StocksPage = require('@pages/BO/catalog/stocks');
+
+let browser;
+let page;
+let numberOfProducts = 0;
+const productStock = Products.demo_18;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    stocksPage: new StocksPage(page),
+  };
+};
+
+// Update Quantity
+describe('Update Quantity', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO and go to categories page
+  loginCommon.loginBO();
+
+  it('should go to "Catalog>Stocks" page', async function () {
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.catalogParentLink,
+      this.pageObjects.boBasePage.stocksLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.stocksPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.stocksPage.pageTitle);
+  });
+
+  it('should get number of products in list', async function () {
+    numberOfProducts = await this.pageObjects.stocksPage.getNumberOfProductsFromList();
+    await expect(numberOfProducts).to.be.above(0);
+  });
+
+  // Update quantity by setting input value
```

Comment:
Not a great added value if this is just a copy-paste of the `it` below, don't you think ? Maybe explaining more what's happening ?

Reply:
done